### PR TITLE
Add Pretraining Integration with LightlyTrain

### PR DESCRIPTION
**Pull request by liopeer**
[Lightly**Train**](https://github.com/lightly-ai/lightly-train) is a novel framework that let's you pretrain any computer vision model on your unlabeled data, by leveraging distillation from foundation models and using self-supervised learning. It has never been easier for the community to pretrain domain-specific backbones for any YOLO downstream task, such as classification, object detection, and more on their own data.

Getting started with Lightly**Train** and YOLO requires only a few lines of code:
```python
import lightly_train
from ultralytics import YOLO

lightly_train.train(
  out="my_experiment",
  data="my_image_folder",
  model="ultralytics/yolo11s.yaml"
)

model = YOLO("my_experiment/exported_models/exported_last.pt")
model.train(data="mydata.yaml")
```
For a full working sample check out our [Lightly**Train** + YOLO Colab](https://colab.research.google.com/github/lightly-ai/lightly-train/blob/main/examples/notebooks/ultralytics_yolo.ipynb)

## Changes
This PR adds an integration with LightlyTrain to the Ultralytics documentation, which shows how a YOLO11 model can be pre-trained on label-free PASCAL VOC, before using the pre-trained backbone to finetune for the task of object detection using Ultralytics' YOLO framework.

I have read the CLA Document and I sign the CLA

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
Adds a new LightlyTrain integration guide showing how to pretrain YOLO models on unlabeled data and then fine-tune for object detection, plus minor docs navigation and authors updates. ⚡️

### 📊 Key Changes
- New docs page: “LightlyTrain” integration (docs/en/integrations/lightlytrain.md) explaining:
  - Self-supervised/distillation pretraining on unlabeled COCO images
  - Fine-tuning YOLO11 on PASCAL VOC with Ultralytics
  - Example code (CLI and Python), data download, visualization, and results comparison
- Integration listed on the Integrations index and added to mkdocs navigation
- Author metadata updated to include a Lightly contributor

### 🎯 Purpose & Impact
- Unlocks value from unlabeled datasets by guiding users to pretrain YOLO models before fine-tuning, improving accuracy and convergence speed 🚀
- Provides a ready-to-run tutorial (including Colab) that lowers the barrier to adopting self-supervised learning with YOLO11 📘
- Enhances discoverability of LightlyTrain within Ultralytics docs and strengthens ecosystem collaboration 🤝

### CODE REVIEW

**Comment by liopeer** [comment]
I have read the CLA Document and I sign the CLA

**Comment by liopeer** [comment]
recheck

**Comment by yutong-xiang-97** [comment]
I have read the CLA Document and I sign the CLA

**Comment by glenn-jocher** [comment]
Thank you @liopeer and @yutong-xiang-97 for submitting this integration guide for LightlyTrain. This looks like a valuable addition to our documentation that could help users leverage unlabeled data for pretraining YOLO models.

The Ultralytics team welcomes these types of integrations as they help expand our ecosystem and provide additional workflow options for users. We'll review the PR soon to ensure it meets our documentation standards and provides clear instructions for users wanting to implement this pretraining approach.

If you have any questions during the review process, feel free to ask here in the comments. We appreciate your contribution to the YOLO community!

**Comment by liopeer** [comment]
Hi @glenn-jocher! We were wondering: When do you think will your team have time to review this PR? Would be great to receive your feedback!

**Comment by glenn-jocher** [comment]
Hi @liopeer! Thanks for following up. We appreciate your patience with the review process. I'll make sure to prioritize this PR for review within the next few days. 

The integration looks promising and the documentation appears well-structured from my initial glance. We'll provide detailed feedback once we've had a chance to thoroughly review the implementation and ensure it aligns with our integration standards.
